mesa/space.py

```diff
@@ -15,9 +15,15 @@
 # pylint: disable=invalid-name
 
 import itertools
-import numpy as np
 import random
 import math
+import numpy as np
+
+# \begin{mypy stuff}
+from typing import Any, Iterator, List, Tuple, Union
+from .agent import Agent
+Coordinate = Tuple[int, int]
+# \end{mypy stuff}
```

Comment:
Do these comments have any utility in mypy? If not could they be removed?

Reply:
Ok, removing.